pyqir/tests/test_parser.py

```diff
@@ -186,9 +186,11 @@ def test_parser_internals() -> None:
     assert interop_funcs[0].name == func_name
     assert required_num_qubits(interop_funcs[0]) == 6
 
-    attribute = interop_funcs[0].attribute("requiredQubits")
-    assert attribute is not None
-    assert attribute.value == "6"
+    attributes = interop_funcs[0].attributes.func
```

Comment:
We are missing tests for parameters and return values.

Reply:
I added some.